### ISSUE TITLE
A donation has a red arrow, as if it is an expense

### ISSUE THREAD

**Issue by haakonnessjoen** [opencollective/opencollective#1392]
**Describe the bug**
A donation has a red arrow, as if it is an expense

**To Reproduce**

After reloading, I cannot reproduce. But I did take a sceenshot.

It was present on this page:
  https://opencollective.com/companion#budget

**Screenshots**
![image](https://user-images.githubusercontent.com/416667/47857964-3d484680-ddeb-11e8-81d5-f5591ea3ccbd.png)

**Comment by znarf** [comment]
@haakonnessjoen thank you for reporting. It might be difficult to solve if we can't reproduce the issue. Please update the issue if you find a way!

**Comment by znarf** [comment]
Duplicate with https://github.com/opencollective/opencollective/issues/1837

**Comment by piamancini** [comment]
This should be solved in the NCP - can we close @znarf ?

**Comment by znarf** [comment]
No, this is still valid with the New Collective Page, see: https://staging.opencollective.com/babel/transactions

**Comment by piamancini** [comment]
ok then. Let's add it to the bug hunting hackathon

**Comment by piamancini** [comment]
@znarf that's a refund, the arrow should be like that.
